src/IssueViz/Models/AnalysisIssueVisualizationConverter.cs

```diff
@@ -48,19 +48,26 @@ public AnalysisIssueVisualizationConverter(IIssueSpanCalculator issueSpanCalcula
             this.issueSpanCalculator = issueSpanCalculator;
         }
 
-        public IAnalysisIssueVisualization Convert(IAnalysisIssue issue, ITextSnapshot textSnapshot)
+        public IAnalysisIssueVisualization Convert(IAnalysisIssueBase issue, ITextSnapshot textSnapshot = null)
         {
-            var span = issueSpanCalculator.CalculateSpan(issue, textSnapshot);
+            var issueSpan = textSnapshot == null
```

Comment:
This is hacky - the real solution should be the separate between IssueViz creation, and span calculations, and this class currently does both. I created #1857 to handle this.